### PR TITLE
RavenDB-13769 Create UI for adding DB IDs to unused DB IDs

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/database/advanced/databaseIDs.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+                this.isForbidden(!accessManager.default.operatorAndAbove());
+                
+                if (this.isForbidden()) {
+                    deferred.resolve({ can: true });
+                } else {
+                    const fetchStatsTask = this.fetchStats();
+                    const fetchUnusedIDsTask = this.fetchUnusedDatabaseIDs();
+
+                    return $.when<any>(fetchStatsTask, fetchUnusedIDsTask)
+                        .then(() => (deferred.resolve({ can: true })))
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/advanced/databaseIDs.ts]: remove ( ) around deferred no need for extra braces

File: src/Raven.Studio/typescript/viewmodels/database/advanced/databaseIDs.ts [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+                        .fail(() => (deferred.resolve({ redirect: appUrl.forStatus(this.activeDatabase()) })));
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/advanced/databaseIDs.ts]: same here

File: src/Raven.Studio/typescript/viewmodels/database/advanced/databaseIDs.ts (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/database/advanced/databaseIDs.ts]: done

File: src/Raven.Studio/typescript/viewmodels/database/advanced/databaseIDs.ts (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/database/advanced/databaseIDs.ts]: done

**Review by ml054 — APPROVED** [review]

**Comment by ppekrol** [comment]
@Danielle9897 Can you rebase?

**Comment by ml054** [comment]
done - please merge

**Review by ppekrol — APPROVED** [review]

**Pull request merged**